npm/npm_import.bzl

```diff
@@ -225,11 +236,15 @@ def npm_import(
             At best omitting this field will make your build non-hermetic.
 
             It is optional to make development easier but should be set before shipping.
+
```

Comment:
Are these formatting changes correct? The other arguments above the diff don't have the blank line...

Reply:
Whoops. Cleaned up.